gecko_client/pchain_subnet_api_test.go

```diff
@@ -0,0 +1,25 @@
+package gecko_client
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCreateSubnet(t *testing.T) {
+	testUnsignedTransactionResult := "1112LA7e8GvkGHDkxZa9Q7kszqvWHooumX5PhqA9NJG7erwXYcwQUPRQyukYX1ncu1DmWvvPNMuivUqvGp1t9M3wys5joqXrXtV2jescQ5AWaUKHiSBUWBRHseMLhGxWNT4Bv6LNVvaaA1ZW33avQBAzz7V84KpKGW7fD3Fz1okxknLgoG"
```

Comment:
Can do this way; I just used the strings demarcated by backticks and copy-pasted exactly what they had in their API docs (see my tests) - seemed easier that way rather than dealing with Sprintf and unmarshalling

Reply:
changed to your way for uniformity, will do this as well throughout